<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Operaciones con Lenguajes</string>
  </property>
  <property name="windowIcon">
   <iconset resource="r/resource.qrc">
    <normaloff>:/r/sigmaLambda.png</normaloff>:/r/sigmaLambda.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Lenguaje Resultante</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Alfabeto</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="alphabetDisplay">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>561</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="resultDisplay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>571</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QDockWidget" name="dockWidget_Language">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>270</y>
      <width>141</width>
      <height>261</height>
     </rect>
    </property>
    <property name="windowTitle">
     <string>Lenguaje</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_3">
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Palabras</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="languageName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="languageButtons">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>200</y>
        <width>131</width>
        <height>27</height>
       </rect>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="words">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>90</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese los simbo-
los separados por 
comas.</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedArgument">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>209</width>
      <height>111</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QTextBrowser" name="languageDisplay_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>201</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="languageList_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>201</width>
        <height>27</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>&lt;Lenguajes&gt;</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QSpinBox" name="argumentN">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>59</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-25</number>
      </property>
      <property name="maximum">
       <number>25</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>6</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>n:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Siendo n el exponente.</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>K:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="argumentK">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>59</width>
        <height>27</height>
       </rect>
      </property>
      <property name="prefix">
       <string/>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>25</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Siendo K la máxima longitud
 de palabra.</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4"/>
   </widget>
   <widget class="QDockWidget" name="dockWidget_Alphabet">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>141</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="windowTitle">
     <string>Alfabeto</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_2">
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>201</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Símbolos</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="AlphabetName">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="alphabetAdvice">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; color:#aa0000;&quot;&gt;Aviso: &lt;/span&gt;Se eliminara &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;el alfabeto actual.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="simbols">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QDialogButtonBox" name="createButtons">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>131</width>
        <height>27</height>
       </rect>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese los simbo-
los separados por 
comas.</string>
      </property>
     </widget>
     <zorder>label_7</zorder>
     <zorder>AlphabetName</zorder>
     <zorder>alphabetAdvice</zorder>
     <zorder>simbols</zorder>
     <zorder>createButtons</zorder>
     <zorder>label_5</zorder>
     <zorder>label_6</zorder>
    </widget>
   </widget>
   <widget class="QPushButton" name="resolveBotton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>201</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QComboBox" name="operations">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Unión</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Intersección</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Diferencia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Concatenación</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Potencia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Estrella Positiva</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Estrella de Kleene</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Complemento</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Inversa</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="languageList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>&lt;Lenguajes&gt;</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="saveLanguageButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Guardar Lenguaje</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="languageDisplay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveAlphabetButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar Alfabeto</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>171</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>256</y>
      <width>20</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/r/save.png&quot; /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>105</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;img src=&quot;:/r/save.png&quot; /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>label_3</zorder>
   <zorder>label</zorder>
   <zorder>alphabetDisplay</zorder>
   <zorder>resultDisplay</zorder>
   <zorder>stackedArgument</zorder>
   <zorder>dockWidget_Alphabet</zorder>
   <zorder>resolveBotton</zorder>
   <zorder>operations</zorder>
   <zorder>languageList</zorder>
   <zorder>saveLanguageButton</zorder>
   <zorder>languageDisplay</zorder>
   <zorder>saveAlphabetButton</zorder>
   <zorder>label_11</zorder>
   <zorder>label_13</zorder>
   <zorder>dockWidget_Language</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="toolAlfabeto"/>
   <addaction name="actionAlphabet_Create"/>
   <addaction name="actionAlphabet_Open"/>
   <addaction name="separator"/>
   <addaction name="toolLenguajes"/>
   <addaction name="actionLanguage_Create"/>
   <addaction name="actionLanguage_Open"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAlfabeto">
    <property name="title">
     <string>Alfabeto</string>
    </property>
    <addaction name="mAlphabetCreate"/>
    <addaction name="mAlphabetOpen"/>
   </widget>
   <widget class="QMenu" name="menuLenguajes">
    <property name="title">
     <string>Lenguajes</string>
    </property>
    <addaction name="mLanguageCreate"/>
    <addaction name="mLanguageOpen"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
   </widget>
   <addaction name="menuAlfabeto"/>
   <addaction name="menuLenguajes"/>
   <addaction name="menuAyuda"/>
  </widget>
  <action name="actionAlphabet_Create">
   <property name="icon">
    <iconset resource="r/resource.qrc">
     <normaloff>:/r/newA.png</normaloff>:/r/newA.png</iconset>
   </property>
   <property name="text">
    <string>Crear Alfabeto</string>
   </property>
   <property name="toolTip">
    <string>Crear Alfabeto</string>
   </property>
  </action>
  <action name="actionLanguage_Create">
   <property name="icon">
    <iconset resource="r/resource.qrc">
     <normaloff>:/r/newL.png</normaloff>:/r/newL.png</iconset>
   </property>
   <property name="text">
    <string>Crear Lenguaje</string>
   </property>
  </action>
  <action name="actionAlphabet_Open">
   <property name="icon">
    <iconset resource="r/resource.qrc">
     <normaloff>:/r/openA.png</normaloff>:/r/openA.png</iconset>
   </property>
   <property name="text">
    <string>Abrir Alfabeto</string>
   </property>
   <property name="toolTip">
    <string>Abrir Alfabeto</string>
   </property>
  </action>
  <action name="actionLanguage_Open">
   <property name="icon">
    <iconset resource="r/resource.qrc">
     <normaloff>:/r/openL.png</normaloff>:/r/openL.png</iconset>
   </property>
   <property name="text">
    <string>Abrir Lenguaje</string>
   </property>
   <property name="toolTip">
    <string>Abrir Lenguaje</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="toolAlfabeto">
   <property name="text">
    <string>Alfabeto:</string>
   </property>
  </action>
  <action name="toolLenguajes">
   <property name="text">
    <string>Lenguajes:</string>
   </property>
  </action>
  <action name="mAlphabetCreate">
   <property name="text">
    <string>Crear</string>
   </property>
  </action>
  <action name="mAlphabetOpen">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="mLanguageCreate">
   <property name="text">
    <string>Crear</string>
   </property>
  </action>
  <action name="mLanguageOpen">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="r/resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>languageList</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>showLanguageIn1(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>languageList_2</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>showLanguageIn2(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resolveBotton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>resolve()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>operations</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>showArgument(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAlphabet_Create</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>showAlphabetCreate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLanguage_Create</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>showLanguageCreate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createButtons</sender>
   <signal>accepted()</signal>
   <receiver>MainWindow</receiver>
   <slot>createAlphabet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>666</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>635</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createButtons</sender>
   <signal>rejected()</signal>
   <receiver>dockWidget_Alphabet</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>714</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>723</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>languageButtons</sender>
   <signal>accepted()</signal>
   <receiver>MainWindow</receiver>
   <slot>createLanguage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>689</x>
     <y>586</y>
    </hint>
    <hint type="destinationlabel">
     <x>621</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>languageButtons</sender>
   <signal>rejected()</signal>
   <receiver>dockWidget_Language</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>711</x>
     <y>575</y>
    </hint>
    <hint type="destinationlabel">
     <x>746</x>
     <y>542</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveLanguageButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveCurrentLanguage1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveAlphabetButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveAlphabet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>471</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAlphabet_Open</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>openAlphabet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLanguage_Open</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>openLanguage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MainWindow</sender>
   <signal>statusChanged(QString)</signal>
   <receiver>statusbar</receiver>
   <slot>showMessage(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>604</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>statusChanged(QString)</signal>
  <slot>openAlphabet()</slot>
  <slot>saveAlphabet()</slot>
  <slot>openLanguage()</slot>
  <slot>saveLanguage()</slot>
  <slot>showLanguageIn1(int)</slot>
  <slot>showLanguageIn2(int)</slot>
  <slot>showArgument(int)</slot>
  <slot>resolve()</slot>
  <slot>createAlphabet()</slot>
  <slot>createLanguage()</slot>
  <slot>showAlphabetCreate()</slot>
  <slot>showLanguageCreate()</slot>
  <slot>saveCurrentLanguage1()</slot>
  <slot>saveResult()</slot>
 </slots>
</ui>
